### DISCUSSION TITLE
pls add support for a second player

### DISCUSSION THREAD

**Discussion by brickeater10** [arianrhodsandlot/retroassembly#16] [Ideas]
Control config only supports the first player currently, it'd be great if it supported the second player as well so I could play local multiplayer.

**Comment by arianrhodsandlot** [comment]
That seems to be a reasonable feature request. As I see multiple controllers should be supported by default. Are you using a keyboard and want to use different keys to control different players, let's say maybe <kbd>w</kbd><kbd>a</kbd><kbd>s</kbd><kbd>d</kbd> for player 1 and <kbd>↑</kbd><kbd>←</kbd><kbd>↓</kbd><kbd>→</kbd> for player 2?

**Reply by brickeater10** [reply]
yeah, that'd be great
